code/modules/psionics/faculties/coercion.dm

```diff
@@ -60,6 +60,21 @@
 		to_chat(user, SPAN_WARNING("\The [target] is in no state for a mind-read."))
 		return TRUE
 
+	var/obj/item/organ/internal/augment/psi/psiaug
```

Comment:
This entire psiaug shit is copypasted over and over, make this into a proc that you use in the commune proc and here

Reply:
Would using `var/obj/item/organ/internal/augment/psi/psiaug = locate() in src` work instead? It would cause potential issues with multiple psiaugs, but uh. Not really a normal use case we need to worry about?

Reply:
Put that in for now